frontend/src/components/App.tsx

```diff
@@ -11,6 +11,7 @@ const App = () => {
     const [streamedContent, setStreamedContent] = useState('');
     const [isStreaming, setIsStreaming] = useState(false);
     const [checkedItems, setCheckedItems] = useState<string[]>([]);
+    const [isDarkMode, setIsDarkMode] = useState(false);
```

Comment:
This gets wiped out everytime the user closes the window, for this to be actually useful I think this should be persisted. Something like localStorage should be fine under a key like `ZIYA_IS_DARK_MODE` to persist the state. 
It can be get and set in a `ThemeContext` perhaps similar to what you see in `FolderContext.tsx` and `ChatContext.tsx`. Without that I think this is more annoying than useful since the user preference keeps getting wiped on on refresh

Reply:
Added dark mode state persistence in newest commit